jobs/JRRFS_UNGRIB

```diff
@@ -23,7 +30,11 @@ if [[ -z "${ENS_INDEX}" ]]; then
 else
   export RUN="ensrrfs"
 fi
-export jobid=${jobid:-${RUN}_ungrib_${TYPE}_${cyc}}
+if [[ ${GROUP_TOTAL_NUM} -gt 1 ]]; then
```

Comment:
I don't mean to hold this PR but it may be good to be consistent about the use of `((` vs `[[` for all changes.

`if (( ${GROUP_TOTAL_NUM} > 1 )); then`